backend/status_codes.md

```diff
@@ -0,0 +1,53 @@
+# Status Codes
+
+Common cases
+--------------
+
+Status codes for common scenarios:
+
+* General:
```

Comment:
I agree, we should include `bad request`, any specific scenario that you want to document? (because malformed and data types errors are being covered by schema via `412 Precondition Failed`)